prql-compiler/src/lib.rs

```diff
@@ -134,6 +135,12 @@ pub enum Target {
     Sql(Option<sql::Dialect>),
 }
 
+impl Default for Target {
+    fn default() -> Self {
+        Self::Sql(Some(sql::Dialect::default()))
```

Comment:
I wonder this should be `Sql(None)` or `Sql(Some(sql::Dialect::default())`?
If not `Sql(None)`, the default behavior is `sql.generic` and the target specified in the script's prql header is ignored.

```suggestion
        Self::Sql(None)
```